src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,21 @@
+.catalog {
+  min-width: calcMaxWidth($card-width, $card-side-gap, $catalog-side-padding);
+  max-width: calcMaxWidth($card-width, $card-side-gap, $catalog-side-padding);
```

Comment:
hmm.. why do we need the same max and min width?)

Reply:
When screen width is less than 1024px, flex wraps cards and only 3 cards stays in a row. If it's okay, I'll remove asap